src/dynarec/rv64/dynarec_rv64_df.c

```diff
@@ -111,10 +111,26 @@ uintptr_t dynarec64_DF(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                         FCVTDL(v1, x1, RD_RNE);
                     }
                     break;
+                case 1:
+                    INST_NAME("FISTTP Ew, ST0");
+                    v1 = x87_get_st(dyn, ninst, x1, x2, 0, EXT_CACHE_ST_F);
+                    addr = geted(dyn, addr, ninst, nextop, &wback, x3, x4, &fixedaddress, rex, NULL, 0, 0);
+                    FCVTWD(x4, v1, RD_RTZ);
```

Comment:
don't we need a check for Invalid Operation to change value to `0x8000` here?

Reply:
Oh, the interpreter seems not doing it on this one, I'll add that on both sides.

Reply:
Ah, it's probably missing there too them. I would expect all conversion to int to react the same way on invalid operations

Reply:
Emmm, can I just use `FR_NV` on this? It will need a lot of opcodes if not.

Reply:
Start with that, and encased the code in a `if(!box64_dynarec_fastround)` yes.

Reply:
I've pushed a new commit, but since this is a `double -> int16` conversion, but we only have `FCVTWD` at hand, it's not accurate, right?

Reply:
Yep, there should be another test when doing 32bits -> 16bits reduction, to check for overflow there too